queries.sql

```diff
@@ -48,3 +48,10 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, SUM(escape_attempts) as total_escape_attempts FROM animals GROUP BY neutered ORDER BY total_escape_attempts DESC;
 SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+SELECT a.name AS Animals_of_Melody_Pong FROM animals AS a JOIN owners AS o ON a.owner_id = o.id WHERE o.full_name = 'Melody Pond';
+SELECT a.name AS Animal_of_type_Pokemon FROM animals AS a JOIN species AS s ON a.species_id = s.id WHERE s.name = 'Pokemon';
+SELECT o.full_name AS name_of_owner, a.name AS Name_of_animal FROM owners AS o LEFT JOIN animals AS a ON o.id = a.owner_id ORDER BY o.full_name;
+SELECT s.name AS Name_of_species, COUNT(a.id) AS Number_of_animals FROM species AS s LEFT JOIN animals AS a ON s.id = a.species_id GROUP BY s.name ORDER BY s.name;
+SELECT a.name AS animal_name FROM owners AS o JOIN animals AS a ON o.id = a.owner_id WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;
+SELECT o.full_name AS owner_name, COUNT(a.id) AS num_animals_owned FROM owners AS o LEFT JOIN animals AS a ON o.id = a.owner_id GROUP BY o.full_name ORDER BY num_animals_owned DESC LIMIT 1;
```

Comment:
- [ ] As mentioned earlier, having insert queries is crucial for verifying the functionality of your select queries. Please make sure to add your [insert queries](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Insert%20the%20following%20data%20into%20the%20species,All%20other%20animals%20are%20Pokemon), run these queries again, and after verifying their accuracy, update the screenshots in the PR description if necessary. 📝🐾📊

Reply:
Hi @shahadat3669 I am finding it difficult to understand what you mean here "As mentioned earlier, having insert queries is crucial for verifying the functionality of your select queries. Please make sure to add your [insert queries](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Insert%20the%20following%20data%20into%20the%20species,All%20other%20animals%20are%20Pokemon), run these queries again, and after verifying their accuracy, update the screenshots in the PR description if necessary. 📝🐾📊"
Please if you could elarate a bit more. I have made some changes I a not very sure if they align with what you mean here.

Reply:
@shahadat3669 is this the insert query that you mean? (image attactehd) I had this in the screenshot. I do not understand when you say having INSERT QUERIES IS CRUCIAL...... I did insert the data but i might have just missed to include the code in the data file. Please let me know if thats what you mean concerning this one. Thanks

![image](https://github.com/enocol/vet_clinic_database/assets/129378066/f08fdf76-8544-4707-a829-9dc26a748e3c)